vscode-lean4/language-configuration.json

```diff
@@ -5,7 +5,13 @@
     },
     "brackets": [
         ["(", ")"],
+        ["`(", ")"],
+        ["``(", ")"],
         ["[", "]"],
+        ["#[", "]"],
+        ["@[", "]"],
+        ["%[", "]"],
+        ["$[", "]"],
```

Comment:
I would also vote to leave this rule out.

Reply:
Removed in 5c13f5c . :)